setup.py

```diff
@@ -30,6 +30,9 @@
         # 'grpc4bmi>=0.2.12,<0.3',
     ],
     packages=find_packages(),
+    package_data={
+        "": ["*.yaml"],
+    },
     include_package_data=True,
```

Comment:
Yaml file is included in bdist_wheel, but missing in sdist.  To also have yaml file in sdist remove include_package_data field.
```suggestion
```